core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/UserApiServiceSpec.scala

```diff
@@ -129,17 +129,19 @@ class UserApiServiceSpec extends ApiServiceSpec {
     }
   }
 
-  it should "list a user's billing projects" in withTestDataApiServices { services =>
+  it should "list a user's billing projects ordered a-z" in withTestDataApiServices { services =>
       Get("/user/billing") ~>
         sealRoute(services.userRoutes) ~>
         check {
           assertResult(StatusCodes.OK) {
             status
           }
 
-          assertResult(Set(RawlsBillingProjectMembership(testData.billingProject.projectName, ProjectRoles.Owner, CreationStatuses.Ready), RawlsBillingProjectMembership(testData.testProject1.projectName, ProjectRoles.User, CreationStatuses.Ready))) {
+          assertResult(
+            List(RawlsBillingProjectMembership(testData.billingProject.projectName, ProjectRoles.Owner, CreationStatuses.Ready),
+              RawlsBillingProjectMembership(testData.testProject1.projectName, ProjectRoles.User, CreationStatuses.Ready)).sortWith((l, r) => l.projectName.value < r.projectName.value)) {
```

Comment:
It seems like a bit of a tautology to assert with the same logic as the functional code.

You could e.g. figure out what the alphabetical first and last projects are in the test data, and then assert that those are the first and last elements of the list.

Reply:
Good point! Thanks for throwing me a hint as well.

Reply:
(and make sure the test data is _not_ alphabetical so it's really testing something)

Reply:
@aednichols would you mind taking a look at my most recent commit?